src/application/engine/entities/specializations/Specializations.java

```diff
@@ -0,0 +1,47 @@
+package application.engine.entities.specializations;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public enum Specializations
+{
+    ROLLER(1), BLASTER(2), HEAVY(3);
+    private int value;
+ 
+    private static Map<Integer, Specializations> values = new HashMap<>();
+ 
+    static
+    {
+ 
+        values.put(1, ROLLER);
```

Comment:
for the sake of code maintenance and correctness please change this to
`values.put(ROLLER.value, ROLLER)`
or
`values.put(ROLLER.getValue(), ROLLER)`